dotcom-rendering/src/web/lib/dynamicSlices.tsx

```diff
@@ -34,24 +34,30 @@ export const Card100 = ({
 	containerPalette?: DCRContainerPalette;
 	containerType?: DCRContainerType;
 	showAge?: boolean;
-}) => (
-	<UL>
-		<LI padSides={true}>
-			<FrontCard
-				trail={cards[0]}
-				containerPalette={containerPalette}
-				containerType={containerType}
-				showAge={showAge}
-				imagePosition="right"
-				imagePositionOnMobile="top"
-				imageSize="jumbo"
-				headlineSize="huge"
-				headlineSizeOnMobile="large"
-				trailText={cards[0].trailText}
-			/>
-		</LI>
-	</UL>
-);
+}) => {
+	const primaries = cards.slice(0, 1);
+
+	return (
+		<UL>
+			{primaries.map((trail) => (
```

Comment:
I would probably opt for an inline slice, here, or keep “primaries” singular.

```suggestion
			{card.slice(0, 1).map((trail) => (
```